install.sh

```diff
@@ -0,0 +1,188 @@
+#!/bin/sh
+
+set -e
+
+DOTENV_LINTER="dotenv-linter"
+DOTENV_LINTER_REPO="${DOTENV_LINTER}/${DOTENV_LINTER}"
+DOTENV_LINTER_GITHUB="https://github.com/${DOTENV_LINTER_REPO}"
+DOTENV_LINTER_RELEASES="${DOTENV_LINTER_GITHUB}/releases"
+
+usage() {
+    this=$1
+    cat 1>&2 <<EOF
+$this: download binaries for dotenv-linter
+
+USAGE:
+    $this [FLAGS] [OPTIONS] <tag>
+
+FLAGS:
+    -h, --help      Prints help information
+
+OPTIONS:
+    -b, --bindir <DIR_PATH>     Sets bindir or installation directory, Defaults to ./bin
+
+ARGS:
+    <tag>       is a tag from ${DOTENV_LINTER_RELEASES}. If tag is missing, then the latest will be used.
+EOF
+    exit 2
+}
+
+parse_args() {
+  BINDIR=${BINDIR:-./bin}
+  while [ "$#" -gt 0 ]; do
+    case $1 in
+        -h|--help)
+            usage "$0"
+            shift #past argument
+            ;;
+        -b|--bindir)
+            BINDIR="$2"
+            shift # past argument
+            shift # past value
+            ;;
+        *) TAG=$1
+            shift # past argument
+            ;;
+    esac
+  done
+}
+
+main() {
+    parse_args "$@"
+
+    require_cmd uname
+    require_cmd mktemp
+    require_cmd grep
+    require_cmd cat
+    require_cmd rm
+
+    get_architecture || return 1
+    _arch="$RETVAL"
+
+    case $_arch in
+        win-*) _ext=".zip" ;;
+        *) _ext=".tar.gz" ;;
+    esac
+
+    _archive_name="${DOTENV_LINTER}-${_arch}${_ext}"
+
+    if [ -z "${TAG}" ]; then
+        println "The latest version will be installed."
```

Comment:
I'd like to know which version will be installed 🙂

Reply:
> I'd like to know which version will be installed 🙂

In that case, later, I will return the function of getting the latest version using the github API :)

I deleted it just to reduce the number of requests, to one